packages/ra-ui-materialui/src/layout/TopToolbar.tsx

```diff
@@ -20,20 +20,20 @@ const StyledToolbar = styled(Toolbar, {
     display: 'flex',
     justifyContent: 'flex-end',
     alignItems: 'flex-end',
-    paddingBottom: theme.spacing(1),
     gap: theme.spacing(1),
-    [theme.breakpoints.up('sm')]: {
-        minHeight: theme.spacing(6),
-    },
-    [theme.breakpoints.up('xs')]: {
-        paddingLeft: 0,
-        paddingRight: 0,
+    whiteSpace: 'nowrap',
+    flex: '0 1 auto',
+    minHeight: 'unset !important',
```

Comment:
Can't we avoid this?

Reply:
I removed that, as agreed,  toolbar keeps (again) consistent height on List, Edit and View templates